### PR TITLE
Don't parse unexported scrapes

### PR DESCRIPTION
**Pull request by x13n**

### CODE REVIEW

File: prometheus-to-sd/main.go [hunk 1]
```diff
@@ -160,6 +140,17 @@ func readAndPushDataToStackdriver(stackdriverService *v3.Service, gceConf *confi
 	exportTicker := time.Tick(*exportInterval)
 
 	for range time.Tick(*scrapeInterval) {
+		select {
```
Comment (loburm) [inline prometheus-to-sd/main.go]: It's not very clear, why you have moved this code to begin of loop.

File: prometheus-to-sd/translator/translator.go [hunk 2]
```diff
@@ -42,54 +42,63 @@ var supportedMetricTypes = map[dto.MetricType]bool{
...
-	for name, metric := range t.metricFamilies {
+	defer func() { t.batch = nil }()
+	metricFamilies, err := t.batch.metrics.Build(t.commonConfig, t.sourceConfig, t.cache)
+	if err != nil {
+		return ts, err
+	}
+	// Get start time before whitelisting, because process start time
+	// metric is likely not to be whitelisted.
+	startTime := getStartTime(metricFamilies)
+	fmt.Printf("4. len(metricFamilies): %v\n", len(metricFamilies))
```
Comment (loburm) [inline prometheus-to-sd/translator/translator.go]: Is it some debugging leftovers?

Comment (x13n) [reply prometheus-to-sd/translator/translator.go]: Yup, thanks for catching!

File: prometheus-to-sd/main.go (hunk 1 above)
Comment (x13n) [reply prometheus-to-sd/main.go]: Scraping errors could delay exporting, added a comment.

**Pull request merged**